server.py

```
@@ -365,8 +381,15 @@
 
 # health check endpoint
 @app.get(config["observability"]["health_check"]["endpoint"])
-async def health():
-    return {"status": "ok", "timestamp": time.time(), "version": __version__}
+async def health(request: Request):
```

Ruff (0.13.1)

Unused function argument: `request`

(ARG001)